enterprise/backend/src/metabase_enterprise/serialization/serialize.clj

```diff
@@ -15,24 +15,27 @@
             [metabase.models.dimension :refer [Dimension]]
             [metabase.models.field :as field :refer [Field]]
             [metabase.models.metric :refer [Metric]]
+            [metabase.models.native-query-snippet :refer [NativeQuerySnippet]]
             [metabase.models.pulse :refer [Pulse]]
             [metabase.models.pulse-card :refer [PulseCard]]
             [metabase.models.pulse-channel :refer [PulseChannel]]
             [metabase.models.segment :refer [Segment]]
             [metabase.models.table :refer [Table]]
             [metabase.models.user :refer [User]]
+            [metabase.shared.models.visualization-settings :as mb.viz]
             [metabase.util :as u]
             [toucan.db :as db]))
 
 (def ^:const ^Long serialization-protocol-version
   "Current serialization protocol version.
 
   This gets stored with each dump, so we can correctly recover old dumps."
-  1)
+  ;; version 2 - start adding namespace portion to /collections/ paths
```

Comment:
we probably don't have a convention and official docs for the serialization protocol, but assuming this is strictly a superset and backwards compatible (?) then would 1.1 be more appropriate?

Reply:
This is a big enough change that I think a bump is warranted (even if it's not used anywhere).  And this field is a `Long` so we can't do decimals.